src/reference/torii-endpoints.md

```diff
@@ -1,69 +1,167 @@
 # Torii Endpoints
 
-<!-- TODO: write some intro, maybe outline the contents. -->
+::: tip Note
+
+Messages for certain `TORII` operations are encoded with Parity <abbr title="Simple Concatenated Aggregate Little-Endian">SCALE</abbr> Codec commonly used with the [Parity Substrate](https://www.parity.io/technologies/substrate/) blockchain framework, and other blockchains utilizing it.
+
+For more information on Parity SCALE Codec, see the [Substrate Documentation: Type encoding (SCALE)](https://docs.substrate.io/reference/scale-codec/) article and its [official GitHub repository](https://github.com/paritytech/parity-scale-codec).
+
+<!-- TODO: link to our own article about SCALE, once it is written; Issue: https://github.com/hyperledger/iroha-2-docs/issues/367 -->
+
+:::
+
+`TORII` is the Iroha 2 module in charge of handling incoming and outgoing connections. It is used to receive, accept and route incoming instructions, and HTTP queries, as well as run-time configuration updates.
+
+To establish two-way communication with the `TORII` endpoints, the following addresses must be specified in the Iroha 2 configuration files:
+
+1. In the `configs/client_cli/config.json` client configuration file:
+   - `TORII_API_URL` — connects to the `TORII` module responsible for handling incoming and outgoing connections.\
+   This address is the same as the `API_URL` address in the `configs/peer/config.json` peer configuration file.
+   - `TORII_TELEMETRY_URL` — connects to the [Prometheus](https://prometheus.io/) endpoint address that is used as a [metrics](../guide/advanced/metrics.md) tool to monitor the network performance.
+
+   ::: info
+
+   To learn more, see [Client Configuration > Iroha Public Addresses](../guide/configure/client-configuration.md#iroha-public-addresses).
+
+   :::
+
+2. In the `configs/peer/config.json` peer configuration file:
+   - `API_URL` — connects to the `TORII` module responsible for handling incoming and outgoing connections.\
+   This address is the same as the `TORII_API_URL` address in the `configs/client_cli/config.json` client configuration file.
+   - `TELEMETRY_URL` — connects to the [Prometheus](https://prometheus.io/) endpoint address that is used as a [metrics](../guide/advanced/metrics.md) tool to monitor the network performance.
+
+   ::: info
+
+   To learn more, see [Peer Configuration > Iroha Public Addresses](../guide/configure/peer-configuration.html#iroha-public-addresses).
+
+   :::
 
 ## API Version
```

Comment:
It is worth mentioning that this endpoint is available if `telemetry` feature is enabled in Iroha.

It applies to other endpoints as well:

- metrics
- status

And there is also `schema` feature which enables "schema" endpoint.

### What are the features?

Those are flag to compile Iroha with different stuff inside. By default all of them are enabled. Disabling them might be beneficial for optimisation.

I think we should also document features somewhere in the reference, in a different topic.

Reply:
Created a new issue for tracking: #465